apiclient/types/knowledgesource.go

```diff
@@ -36,12 +36,16 @@ type KnowledgeSource struct {
 	LastSyncStartTime       *Time                `json:"lastSyncStartTime,omitempty"`
 	LastSyncEndTime         *Time                `json:"lastSyncEndTime,omitempty"`
 	LastRunID               string               `json:"lastRunID,omitempty"`
+	FilePathPrefixInclude   []string             `json:"filePathPrefixInclude,omitempty"`
+	FilePathPrefixExclude   []string             `json:"filePathPrefixExclude,omitempty"`
```

Comment:
Do we need this both here and `KnowledgeSourceManifest`? Seems like just in `KnowledgeSourceManifest` is enough.